src/View/Sidebar/NetworkListBox.vala

```diff
@@ -70,12 +98,15 @@ public class Sidebar.NetworkListBox : Gtk.ListBox, Sidebar.SidebarListInterface
 
         string scheme = Uri.parse_scheme (mount.get_root ().get_uri ());
 
-        /* Some non-native schemes are still local e.g. mtp, ptp, gphoto2 */
+        /* Some non-native schemes are still local e.g. mtp, ptp, gphoto2.  These are shown in the Device ListBox */
         if ("smb ftp sftp afp dav davs".contains (scheme)) {
```

Comment:
`var scheme =` should do it. Watch out, `Uri.parse_scheme` may return `null`, so we should probably check this in the if below:

```
if (scheme != null && ..
```

Reply:
Done